main.tf

```diff
@@ -4,78 +4,76 @@ locals {
 
-  min_master_version = "${var.min_master_version}"
+  min_master_version = var.min_master_version
   enable_legacy_abac = false
 
-  network    = "${var.network == "" ? local.network_name : var.network}"
-  subnetwork = "${google_compute_subnetwork.nodes-subnet.self_link}"
-  project    = "${var.project}"
+  network    = var.network == "" ? local.network_name : var.network
+  subnetwork = google_compute_subnetwork.nodes-subnet.self_link
+  project    = var.project
 
   ip_allocation_policy {
-    cluster_secondary_range_name  = "${google_compute_subnetwork.nodes-subnet.secondary_ip_range.0.range_name}"
-    services_secondary_range_name = "${google_compute_subnetwork.nodes-subnet.secondary_ip_range.1.range_name}"
+    cluster_secondary_range_name  = google_compute_subnetwork.nodes-subnet.secondary_ip_range[0].range_name
+    services_secondary_range_name = google_compute_subnetwork.nodes-subnet.secondary_ip_range[1].range_name
   }
 
   lifecycle {
-    ignore_changes = ["subnetwork"]
+    ignore_changes = [subnetwork]
   }
 
   initial_node_count       = 1
   remove_default_node_pool = true
 }
 
 resource "google_container_cluster" "primary-regional" {
-  name  = "${var.name}"
-  count = "${var.regional_cluster && !var.nat_enabled  ? 1 : 0 }"
+  name  = var.name
+  count = var.regional_cluster && false == var.nat_enabled ? 1 : 0
 
-  location = "${var.region}"
+  location = var.region
 
-  min_master_version = "${var.min_master_version}"
+  min_master_version = var.min_master_version
   enable_legacy_abac = false
 
-  network    = "${var.network == "" ? local.network_name : var.network}"
-  subnetwork = "${google_compute_subnetwork.nodes-subnet.self_link}"
-  project    = "${var.project}"
+  network    = var.network == "" ? local.network_name : var.network
+  subnetwork = google_compute_subnetwork.nodes-subnet.self_link
+  project    = var.project
 
   ip_allocation_policy {
-    cluster_secondary_range_name  = "${google_compute_subnetwork.nodes-subnet.secondary_ip_range.0.range_name}"
-    services_secondary_range_name = "${google_compute_subnetwork.nodes-subnet.secondary_ip_range.1.range_name}"
+    cluster_secondary_range_name  = google_compute_subnetwork.nodes-subnet.secondary_ip_range[0].range_name
+    services_secondary_range_name = google_compute_subnetwork.nodes-subnet.secondary_ip_range[1].range_name
   }
 
-  node_locations = [
-    "${formatlist("%s-%s", var.region, var.zones)}",
-  ]
+  node_locations = formatlist("%s-%s", var.region, var.zones)
 
   lifecycle {
-    ignore_changes = ["subnetwork"]
+    ignore_changes = [subnetwork]
   }
 
   initial_node_count       = 1
   remove_default_node_pool = true
 }
 
 resource "google_container_cluster" "primary-nat" {
-  name = "${var.name}"
+  name = var.name
 
   location = "${var.region}-${var.zones[0]}"
-  count    = "${!var.regional_cluster && var.nat_enabled  ? 1 : 0 }"
+  count    = false == var.regional_cluster && var.nat_enabled ? 1 : 0
```

Comment:
`var.regional_cluster == false` - convention

Reply:
as above, I will fix it.